api/services/recipeform.service.js

```diff
@@ -1,27 +1,24 @@
-const measures = require('../measurements');
-
-const toMap = measures.measuresToMap;
-const toFind = measures.measuresToFind;
+const {measuresToMap, measuresToFind} = require("../measurements");
 
 exports.buildUrl = (name) => {
-    return name.toLowerCase().replace(/\s/g, '-');
+    return name.toLowerCase().replace(/\s/g, "-");
 }
 
 //replace/update when tags library is found
 exports.formatKeywords = (keywords) => {
-    let formattedKeywords = keywords.replace(',', '').split(' ');
+    let formattedKeywords = keywords.replace(",", "").split(" ");
     return formattedKeywords;
 }
 
 exports.formatDirections = (directions) => {
-    let formattedDirections = directions.split('\n');
+    let formattedDirections = directions.split("\n");
     return formattedDirections;
 }
 
 exports.formatIngredients = (ingredients) => {
     const ingredientArray = ingredients.split("\n");
     let ingredientObj = [];
-    let measuresToFind = Object.keys(toFind);
+    let toFind = Object.keys(measuresToFind);
```

Comment:
Change this to `measureToFindKeys` to give a better understanding of what's exactly in this variable